lib/tasks/sync_countries.rake

```diff
@@ -0,0 +1,32 @@
+namespace :sync do
+  task :countries => :environment do
+    response = HTTParty.get('http://speciesplus.net/api/v1/geo_entities')
```

Comment:
better to use `/api/v1/geo_entities?geo_entity_types_set=3`, as that will filter out non-current entities